sources/core/Stride.Core.Design/Editor/EditorInfo.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Stride contributors (https://stride3d.net)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System;
+
+namespace Stride.Core.Editor
+{
+    public class EditorInfo
```

Comment:
This is info about the IDE, not stide's editor. You're in ``Stride.Core.Editor`` so the distinction is important